pom.xml

```diff
@@ -759,6 +760,11 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```

Comment:
Please don't throw this in a random location. The `commons-*` dependencies are all together so place with them.

Reply:
Changed in most recent patch.